core/client/app/controllers/setup.js

```diff
@@ -15,6 +15,11 @@ export default Ember.Controller.extend(ValidationEngine, {
     ghostPaths: Ember.inject.service('ghost-paths'),
     notifications: Ember.inject.service(),
 
+    stepOne: true,
```

Comment:
There is an opportunity here to leverage Ember's built-in [computed.equal](http://emberjs.com/api/classes/Ember.computed.html#method_equal) helper to simplify the code a bit, eg:

```
currentStep: 'one',

stepOne: Ember.computed.equal('currentStep', 'one'),
stepTwo: Ember.computed.equal('currentStep', 'two'),
stepThree: Ember.computed.equal('currentStep', 'three'),

actions: {
    toStepOne: function () { this.set('currentStep', 'one'); }
    ...
}
```